commit 6c1db4afb98e1718fa237c6310170b5980d4af81
Author: Francesco Riosa <francesco+kde@pnpitalia.it>
Date:   Wed May 16 00:09:09 2012 +0200

    dkCmsTakeProfileID allocate right size
    
    Thanks Kevin Kofler and Nucleo(Fedora) for squashing it.
    BUG: 299886

diff --git a/libs/dklcms/digikam-lcms.cpp b/libs/dklcms/digikam-lcms.cpp
index 468eab6..f8542d6 100644
--- a/libs/dklcms/digikam-lcms.cpp
+++ b/libs/dklcms/digikam-lcms.cpp
@@ -491,7 +491,7 @@ LCMSAPI DWORD LCMSEXPORT dkCmsTakeHeaderFlags(cmsHPROFILE hProfile)
 
 LCMSAPI const BYTE* LCMSEXPORT dkCmsTakeProfileID(cmsHPROFILE hProfile)
 {
-    cmsUInt8Number* ProfileID = new cmsUInt8Number();
+    cmsUInt8Number* ProfileID = new cmsUInt8Number[16];
     cmsGetHeaderProfileID(hProfile, ProfileID);
     return static_cast<BYTE*>( ProfileID );
 }
